config/initializers/monkeypatches/download.rb

```diff
@@ -0,0 +1,43 @@
+module Geoblacklight
+  class Download
```

Comment:
Suggestion: My assumption reading this is that all these methods are overridden and customized completely, if that's not true can we have some comments sharing which are overridden and how?

Reply:
This is an older monkey patch that I moved to this location. Yes we can add comments.

Reply:
Actually, already has a comment. I'll add link to the file that was overridden.

Reply:
Link added.